src/serializer/ResidualHeapSerializer.js

```diff
@@ -151,11 +151,16 @@ export class ResidualHeapSerializer {
     properties: Map<string, PropertyBinding> = obj.properties,
     objectPrototypeAlreadyEstablished: boolean = false
   ) {
-    /*
-    for (let symbol of obj.symbols.keys()) {
-      // TODO #22: serialize symbols
+    //inject symbols
+    for (let [symbol, propertyBinding] of obj.symbols) {
+      invariant(propertyBinding.descriptor);
+      let desc = propertyBinding.descriptor;
+      // TODO:#22 move to emit property
```

Comment:
I suspect that deletion of symbols fully works. So there should be a statement like `if (desc === undefined) continue; //deleted`, as well as a test case that exercises deletion of a property.